src/components/DetailsContainer/DetailsContainer.css

```diff
@@ -0,0 +1,185 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;400;700&display=swap');
```

Comment:
Inspiring CSS efforts, go Irmak!!!! 🥳 Send our regards to Keith. Tell him "Regards."